.pre-commit-config.yaml

```diff
@@ -97,7 +97,7 @@ repos:
   - id: detect-private-key
     exclude: ^examples/
 - repo: https://github.com/asottile/pyupgrade
-  rev: 'v3.17.0'
+  rev: 'v3.18.0'
   hooks:
   - id: pyupgrade
     args: ['--py37-plus']
```

Comment:
We could probably bump this up soon

Reply:
I think it currently gets delayed one version, otherwise it tries to remove the version check in setup.py, if I remember correctly. I expect we'll get round to migrating to ruff in the near future, which should resolve any issues.